<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Gráfico Interativo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="base_horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="menuFunc_groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>136</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Menu de funções</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="menuFunc_verticalLayout">
           <item>
            <widget class="QGroupBox" name="objetos_groupBox">
             <property name="minimumSize">
              <size>
               <width>108</width>
               <height>150</height>
              </size>
             </property>
             <property name="title">
              <string>Objetos</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QListWidget" name="objects">
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="salvarObjeto">
                <property name="text">
                 <string>Salvar Objetos</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="window_groupBox">
             <property name="title">
              <string>Window</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="window_verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="passo_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="passo_label">
                    <property name="text">
                     <string>passo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="passo_field">
                    <property name="text">
                     <string>100</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="percent_label">
                    <property name="text">
                     <string>%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="in_window">
                    <property name="text">
                     <string>In</string>
                    </property>
                    <property name="autoDefault">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="out_window">
                    <property name="text">
                     <string>Out</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QPushButton" name="up_window">
                    <property name="text">
                     <string>Up</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="left_window">
                      <property name="text">
                       <string>Left</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="right_window">
                      <property name="text">
                       <string>Right</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="down_window">
                    <property name="text">
                     <string>Down</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="zoom_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="zoom_label">
                    <property name="text">
                     <string>Zoom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="plusZoomWindow">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="minusZoomWindow">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QComboBox" name="windowsBox"/>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Rotacionar</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="rotateVUp">
                       <property name="text">
                        <string>z</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateVUpMinus">
                       <property name="text">
                        <string>&lt;-</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateVUpPlus">
                       <property name="text">
                        <string>-&gt;</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>x</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateXMinus">
                       <property name="text">
                        <string>&lt;-</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateXPlus">
                       <property name="text">
                        <string>-&gt;</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>y</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateYMinus">
                       <property name="text">
                        <string>&lt;-</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="rotateYPlus">
                       <property name="text">
                        <string>-&gt;</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="viewPort_verticalLayout">
        <item>
         <widget class="QGraphicsView" name="viewPort_graphicsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sceneRect">
           <rectf>
            <x>0.000000000000000</x>
            <y>0.000000000000000</y>
            <width>0.000000000000000</width>
            <height>0.000000000000000</height>
           </rectf>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="transformationAnchor">
           <enum>QGraphicsView::NoAnchor</enum>
          </property>
          <property name="viewportUpdateMode">
           <enum>QGraphicsView::MinimalViewportUpdate</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_label">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="status_textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>528</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNovo">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="novoObjeto"/>
    <addaction name="abrirXml"/>
    <addaction name="salvarXml"/>
   </widget>
   <addaction name="menuNovo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="novoObjeto">
   <property name="text">
    <string>&amp;Novo Objeto</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="abrirXml">
   <property name="text">
    <string>Carregar cena / objeto</string>
   </property>
  </action>
  <action name="salvarXml">
   <property name="text">
    <string>Salvar cena</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
